api/v1alpha1/clusterqueue_types.go

```diff
@@ -189,48 +170,17 @@ type Resource struct {
 	// - name: nvidia.com/gpus
 	//   - name: k80
 	//     quota:
-	//       guaranteed: 10
-	//     labels:
-	//       cloud.provider.com/accelerator: nvidia-tesla-k80
+	//       min: 10
 	//   - name: p100
 	//     quota:
-	//       guaranteed: 10
-	//     labels:
-	//       cloud.provider.com/accelerator: nvidia-tesla-p100
+	//       min: 10
 	//
 	// The flavors are evaluated in order, selecting the first to satisfy a
 	// workload’s requirements. Also the quantities are additive, in the example
 	// above the GPU quota in total is 20 (10 k80 + 10 p100).
 	// A workload is limited to the selected type by converting the labels to a node
-	// selector that gets injected into the workload. ​​This list can’t be empty, at
-	// least one must exist.
-	//
-	// Note that a workload’s node affinity/selector constraints are evaluated
```

Comment:
It is worth having this in the docs

Reply:
It is https://github.com/kubernetes-sigs/kueue/blob/main/docs/concepts/cluster_queue.md#resourceflavor-labels